surface_dynamics/flat_surfaces/abelian_strata.py

```diff
@@ -29,7 +29,7 @@
 The inverse operation (pass from an interval exchange transformation to
 the connected component) is partially written in [KonZor03]_ and
 simply named here
-:meth:`~surface_dynamics.interval_exchanges.template.PermutationIET.components`.
+?
```

Comment:
I replaced some things that sphinx was unhappy about with `?`. Please have a look.

Reply:
Could you be more specific here?

Reply:
Sorry, what is unclear exactly? This one change or all the `?`?
If you look at the diff, there are lots of `?` where Sphinx said that it could not resolve something.